cmd/libsnap-confine-private/locking.c

```diff
@@ -84,7 +84,7 @@ void sc_disable_sanity_timeout(void)
 
 static const char *sc_lock_dir = SC_LOCK_DIR;
 
-int sc_lock(const char *scope)
+static int sc_lock_generic(const char *scope, int uid)
```

Comment:
why `int uid`? `int` is signed, and `uids` are `uid_t`, which are unsigned 32-bit quantities.

Reply:
I think you are right. I'll change this.